app/commands/recent.py

```diff
@@ -107,3 +114,61 @@ async def recent(context: Context):
         embed=embed, reference=context.message, mention_author=True,
         view=replay
     )
+
+
+def get_difficulty_info(score: DBScore) -> Tuple[float, float, float]:
+    # fc_pp, star rating, approximate lazer sr (rx/ap)
+    beatmap_file = app.session.storage.get_beatmap(score.beatmap_id)
+
+    if not beatmap_file:
+        app.session.logger.error(
+            f'pp calculation failed: Beatmap file was not found! ({score.user_id})'
+        )
+        return 0.0, 0.0, 0.0
+
+    mods = Mods(score.mods)
+
+    if Mods.Nightcore in mods and not Mods.DoubleTime in mods:
+        # NC somehow only appears with DT enabled at the same time...?
+        # https://github.com/ppy/osu-api/wiki#mods
+        mods |= Mods.DoubleTime
+
+    if Mods.Perfect in mods and not Mods.SuddenDeath in mods:
+        # The same seems to be the case for PF & SD
```

Comment:
don't think so?

Reply:
This is likely an old client bug. Happened to me on score submission + multiplayer...